canonical-websites/snapcraft.io
# Updated snap details page title

Fixes #31 

# QA
- ./run
- Go to any snap page (http://localhost:8022/dwarf-fortress/)
- Check page title (in browser tab)
